setup.cfg

```
@@ -0,0 +1,19 @@
+[metadata]
+name = xtarfile
+version = file: version.txt
+url = https://github.com/ascoderu/xtarfile
+download_url = https://pypi.python.org/pypi/xtarfile
+author = Clemens Wolff
+author_email = clemens.wolff+pypi@gmail.com
+description = Wrapper around tarfile with support for more compression formats.
+long_description = file: README.rst
+license = Apache Software License
+classifiers = License :: OSI Approved :: Apache Software License, Programming Language :: Python :: 3.4, Programming Language :: Python :: 3.5, Programming Language :: Python :: 3.6, Topic :: Utilities
```

Given that CI now runs for Python 3.6+ and that this is also the declared minimum Python version, I would suggest to update the tags too:

```suggestion
classifiers = License :: OSI Approved :: Apache Software License, Programming Language :: Python :: 3.6, Programming Language :: Python :: 3.7, Programming Language :: Python :: 3.8, Programming Language :: Python :: 3.9, Topic :: Utilities
```

Also: is it possible to split this list over multiple lines to improve readability?

---

Fixed in 4b0bb13